apps/files_external/tests/Storage/SftpTest.php

```
@@ -133,4 +134,37 @@ public function configProvider() {
 			],
 		];
 	}
+
+	public function testAuthenticated(): void {
+		$this->assertTrue($this->instance->getConnection()->isAuthenticated());
+	}
```

Should not be necessary, but might be nice to add to `setUp` so any test fails early and not with a confusing error message that is not related to the actual test.

---

It makes sense to just remove it anyway since `setUp` already has a `mkdir()` call which requires the client to be authenticated.